### PR TITLE
Fix couple of issues regarding Graph Overview

### LINKED ISSUE #1221: Couple of issues regarding Graph Overview
Env: Ubuntu 16.04 LTS

1. Overview does not update its contents when seek changed with Graph closing.
2. Overview will be gone completely after 
---
how-to:
1. go to disas
2. seek to somewhere else where no function is detected
3. go to Graph
4. seek to somewhere else where Graph exists

gif would tell you more anyway so let me just attach gifs for them quickly.

gifrecord_2019-02-27_193421
gifrecord_2019-02-27_193451

### PR DESCRIPTION
**Pull request by Vane11ope**
fix #1221

### CODE REVIEW

**Comment by Vane11ope** [comment]
@xarkes weird. it worked well last time i checked. i need to check this again later, thanks for the feedback :)

**Comment by Vane11ope** [comment]
@xarkes it absolutely works for me.

**Comment by Vane11ope** [comment]
@xarkes is this probably for "open->save layout->close->reopen->everything messed up" issue of the layout? then that's not related to this PR, just skip the first phase and keep using it. if overview still not synced with anything then... idk

**Comment by xarkes** [comment]
I will try again later tonight.

**Comment by ITAYC0HEN** [comment]
issue: moving Graph Overview to other place will break sync

**Comment by Vane11ope** [comment]
Done.

File: src/widgets/OverviewWidget.h [hunk 1]
```diff
@@ -31,12 +31,20 @@ private slots:
      * \brief update the overview
      */
     void updateContents();
+    /*
+     * \brief override closeEvent to emit graphClose
```
Comment (xarkes) [inline src/widgets/OverviewWidget.h]: Please use the new documentation format that is doxygen style:
```
/**
 * @brief blabla
 */
```

Comment (Vane11ope) [reply src/widgets/OverviewWidget.h]: .............................i used this format first and itay corrected me this \brief way..............................

which one is right exactly

Comment (xarkes) [reply src/widgets/OverviewWidget.h]: We took the decision to use the one I just told you :-) Sorry that we gave you contradictory indications :P

Comment (xarkes) [reply src/widgets/OverviewWidget.h]: Today basically so you were doing good :+1:

Comment (Vane11ope) [reply src/widgets/OverviewWidget.h]: lol yeah things like this happen i know i was just messing with you. fix them right away hang on.

Comment (Vane11ope) [reply src/widgets/OverviewWidget.h]: you want me to fix all of the comments? then I think we should separate the PR?

Comment (xarkes) [reply src/widgets/OverviewWidget.h]: I fixed all the comments on master. Just rebase ;)

Comment (Vane11ope) [reply src/widgets/OverviewWidget.h]: Done.

**Pull request merged**